openlibrary/templates/books/edit/about.html

```diff
@@ -1,5 +1,23 @@
 $def with (work)
 
+$# Render the subject input field
+$jsdef render_subject_field(name, facet, data):
+    <div class="input">
+        $code:
+            subjects = []
+            for s in data:
+                s = s.replace('"', '""')
+                subjects.append('"' + s + '"' if ',' in s else s)
+            subject_str = ', '.join(subjects)
+            rows = len(subject_str) // 95 + 1
+        <textarea name="work--$name" id="about-subjects-$facet" rows="$rows" wrap="soft" tabindex="0">$subject_str</textarea>
```

Comment:
These `id`s aren't quite the same to what they used to be, but I can't find a spot where the old IDs were used, so let's just remove them. We could then also potentially remove the `facet` parameter here!
```suggestion
        <textarea name="work--$name" rows="$rows" wrap="soft" tabindex="0">$subject_str</textarea>
```